modern-redux-with-redux-toolkit-rtk-and-typescript-64f243c8/notes/03_react-add-redux-and-the-redux-toolkit-rtk-to-an-existing-application.md

```diff
@@ -4,16 +4,15 @@
 
 <TimeStamp start="0:02" end="0:10">
 
-```
-npm install react-redux
-npm install @reduxjs/toolkit
+```bash
+npm install react-redux @reduxjs/toolkit
 ```
```

Comment:
for people who want to copy paste the install, you can do multiple packages at once by comma separating them